docs/DeveloperGuide.md

```diff
@@ -724,29 +724,47 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+### Deleting an animal
 
-### Deleting a person
+1. Deleting an animal while all animals are being shown
 
-1. Deleting a person while all persons are being shown
+   1. Prerequisites: List all animals using the `list` command. Multiple animals in the list.
 
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
-
-   1. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message. Timestamp in the status bar is updated.
+   1. Test case: `delete 123`<br>
+      Expected: The animal with an ID of 123 deleted from the list. Details of the deleted animal shown in the status message. Timestamp in the status bar is updated.
 
    1. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message. Status bar remains the same.
+      Expected: No animal is deleted. Error details shown in the status message. Status bar remains the same.
 
-   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is smaller than the requisite size)<br>
       Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
+1. Deleting an animal while not all animals are being shown
+   
+   1. Prerequisites: Use `find 455` command to list animals that contain 455 in any of their fields. 
+      2 animals are listed: an anaconda named 455 and a lion named Simba with an ID of 455.
+   
+   1. Test case: `delete 455`<br>
+      Expected: Simba with an ID of 455 is deleted from the list. Details of the deleted animal shown in the status message. Timestamp in the status bar is updated.
+
+   1. Test case: `delete 123`<br>
+      Expected: No animal is deleted. Error details shown in the status message. Status bar remains the same.
+
+   1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is smaller than the requisite size)<br>
+      Expected: Similar to previous.
```

Comment:
May have to modify this since delete should work without finding anymore.

Reply:
Deleted